### PR TITLE
Correct training-argument documentation that disagrees with the trainer

### PR DESCRIPTION
**Pull request by raimbekovm**
## summary

- `model-evaluation-insights.md` told readers to set `warmup_epochs=0` when fine-tuning while `finetuning-guide.md` recommended keeping the 3-epoch default. The released checkpoints settle it: stage 2 of the official YOLO26 recipe is itself a fine-tune from Objects365 weights, and its searched `warmup_epochs` is about one epoch for every size — neither value. The recommendation now lives once, at its owner, with its condition and a link to the recipe. The evaluation guide's lede, Conclusion, FAQ and index blurb no longer promise the deleted content.
- `warmup_bias_lr` was documented with default `0.1` while the shipped default `optimizer='auto'` sets it to `0.0`, discarding user-supplied values. Measured on both auto branches, not just AdamW.
- `optimizer` was described as selecting "based on model configuration" in the macro and again in `model-training-tips.md`, eleven lines below a correct statement of the same rule. It selects on the number of training iterations, the wording `build_optimizer`'s docstring already uses.
- `freeze` was documented as accepting an integer or layer indices. A list entry is interpolated into `model.{x}.` and matched as a name prefix, so module names work too. Documented with an example, because the name PyTorch itself reports (`model.23.cv2`) is exactly the one that silently freezes nothing.

## measured

`train_args` read from the released checkpoints:

| checkpoint | starting weights | epochs | `warmup_epochs` |
| --- | --- | --- | --- |
| `yolo26n.pt` | `yolo26n-objv1-150.pt` | 245 | 0.98124 |
| `yolo26s.pt` | `yolo26s-objv1-150.pt` | 70 | 0.98745 |
| `yolo26x.pt` | `yolo26x-objv1-150.pt` | 40 | 0.98745 |
| `yolo26n-objv1-150.pt` | `yolo26n.yaml` (from scratch) | 150 | 1 |

Frozen-parameter counts and optimizer choice on `yolo26n.pt`, captured at `on_train_start`:

```
optimizer='SGD'                      -> SGD    warmup_bias_lr=0.05
optimizer='auto', iterations=5000    -> AdamW  warmup_bias_lr=0.0
optimizer='auto', iterations=50000   -> MuSGD  warmup_bias_lr=0.0

freeze=2                             ->  6/366 frozen
freeze=["23.cv2","23.one2one_cv2"]   -> 48/366 frozen
freeze=["model.23.cv2"]              ->  0/366 frozen
```

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary

Updated training documentation to match the trainer’s implemented behavior and released YOLO26 training recipes.

### 📊 Key Changes

- Fine-tuning guidance now recommends non-zero warmup without requiring the full 3-epoch default, linking to the official YOLO26 recipe’s approximately one-epoch setting.
- Documented that `optimizer='auto'` selects AdamW or MuSGD based on training iterations, and that it overrides `warmup_bias_lr` to `0.0`; explicit optimizer selection is required to use the configured bias warmup value.
- Expanded `freeze` documentation to support module names such as `23.cv2` in addition to layer counts and indices, while clarifying that names should omit the leading `model.` prefix.
- Removed the conflicting no-warmup fine-tuning instructions and updated related evaluation-guide links, FAQ text, conclusion, and index description.

### 🎯 Purpose & Impact

- Users following the documentation will receive training guidance consistent with the trainer, including the actual `optimizer='auto'` and `freeze` behavior.
- Fine-tuning recommendations now point users to the dedicated guide and official YOLO26 recipe instead of instructing them to disable warmup.
- These changes update documentation and configuration comments only; no trainer implementation behavior was changed.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @raimbekovm, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! This automated message confirms your contribution was received, and an Ultralytics engineer will assist with the review. To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_ — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Reviewed all changed documentation and the linked YOLO26 recipe context. The warmup recommendation and links align with the repository’s documented stage-2 values. One low-severity stale-reference issue remains after deleting the former fine-tuning subsection.

<details><summary>💬 Posted 1 inline comment</summary>

- 📝 **LOW** `docs/en/guides/model-evaluation-insights.md:112` Removing the warmup subsection leaves stale references later on this page: the Conclusion still tells readers to “apply the fine-tuning techniques above,” and the mAP FAQ still calls this a “detailed section” with “specific strategies,” although the section now only redirects to the fine-tuning guide and covers image tiling. Update those references to point directly to the fine-tuning guide so the page does not promise guidance that is no longer present.
</details>

File: docs/en/guides/model-evaluation-insights.md [hunk 1]
```diff
@@ -109,33 +109,7 @@ The `results.box.image_metrics` attribute is a per-image dictionary keyed by ima
...
-        # Fine-tune with the warmup phase disabled
-        model.train(data="coco8.yaml", epochs=10, warmup_epochs=0)
-        ```
-
-    === "CLI"
-
-        ```bash
-        yolo detect train model=yolo26n.pt data=coco8.yaml epochs=10 warmup_epochs=0
-        ```
+Fine-tuning takes a pretrained model and adjusts its parameters to improve performance on a specific task or dataset. Also known as model retraining, it lets the model better understand and predict outcomes for the data it will encounter in real-world applications. Based on your evaluation results, you retrain the model on your own data, adjusting the training arguments covered in the [fine-tuning guide](./finetuning-guide.md).
```
Comment (UltralyticsAssistant) [inline docs/en/guides/model-evaluation-insights.md]: 📝 **LOW**: Removing the warmup subsection leaves stale references later on this page: the Conclusion still tells readers to “apply the fine-tuning techniques above,” and the mAP FAQ still calls this a “detailed section” with “specific strategies,” although the section now only redirects to the fine-tuning guide and covers image tiling. Update those references to point directly to the fine-tuning guide so the page does not promise guidance that is no longer present.

**Review by UltralyticsAssistant — DISMISSED** [review]
## 🔍 PR Review 2

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Review 2: The prior stale-reference issue is resolved; the conclusion and mAP FAQ now point directly to the fine-tuning guide, and the removed warmup examples no longer leave misleading references. The updated warmup guidance, recipe link, and guides index description are consistent with the documented YOLO26 stage-2 values. No new bugs or regressions found. LGTM.

**Review by UltralyticsAssistant — DISMISSED** [review]
## 🔍 PR Review 3

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Review 3: The previous stale-reference issue remains resolved. This update consistently documents the effective `warmup_bias_lr` behavior, explains `optimizer='auto'` using training iterations, and links the fine-tuning guidance to the official YOLO26 recipe without introducing behavioral changes or broken references. LGTM.

**Review by UltralyticsAssistant — DISMISSED** [review]
## 🔍 PR Review 4

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Compared with review 3, the documentation changes remain consistent and the previously identified stale references are still resolved. The fine-tuning warmup guidance, optimizer-selection wording, and effective `warmup_bias_lr` behavior match the repository implementation and recipe documentation. No new bugs or regressions found. LGTM.

**Review by UltralyticsAssistant — APPROVED** [review]
## 🔍 PR Review 5

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Since review 4, the previously resolved stale-reference issue remains fixed, and the current documentation changes still match the trainer implementation: optimizer='auto' is iteration-based, warmup_bias_lr is overridden to 0.0 for auto, and freeze supports the documented module-name form. Links and guidance are consistent across the affected guides. No new bugs or regressions found. LGTM.

**Comment by UltralyticsAssistant** [comment]
🎉 **PR [#25957](https://github.com/ultralytics/ultralytics/pull/25957) has been merged—thank you, @raimbekovm!**

> “Alone we can do so little; together we can do so much.” — Helen Keller

This contribution brings the training documentation in line with implemented trainer behavior, clarifies warmup, `optimizer='auto'`, and `freeze` guidance, and adds official YOLO26 training recipes for more accurate, consistent user guidance.
